src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -562,6 +562,43 @@ public void testForceOutput() {
         Assert.assertTrue(actualVC2.get(1).isPolymorphicInSamples());
     }
 
+    @Test
+    public void testForceOutputNonRef() {
+        final File input = getTestFile("ForceOutputNonRef.g.vcf");
+        final File output1 = createTempFile("output", ".vcf");
+
+        final ArgumentsBuilder argsWithAllSites = new ArgumentsBuilder()
+                .addReference(b37Reference)
+                .addVCF(input)
+                .addBooleanArgument(RMSMappingQuality.RMS_MAPPING_QUALITY_OLD_BEHAVIOR_OVERRIDE_ARGUMENT, true)
+                .addOutput(output1);
+
+        Utils.resetRandomGenerator();
+        runCommandLine(argsWithAllSites);
+
+        final List<VariantContext> actualVC = VariantContextTestUtils.getVariantContexts(output1);
+
+        // every site has output
```

Comment:
Should this comment instead say "no site has output"?

Reply:
ok